src/object_properties.cpp

```diff
@@ -71,6 +73,9 @@ void ObjectProperties::serialize(std::ostream &os) const
 	writeF1000(os, weight);
 	writeV3F1000(os, collisionbox.MinEdge);
 	writeV3F1000(os, collisionbox.MaxEdge);
+	writeV3F1000(os, selectionbox.MinEdge);
+	writeV3F1000(os, selectionbox.MaxEdge);
+	writeU8(os, pointable);
```

Comment:
Okay, that makes sense, are there any examples of how to bump the version, another PR perhaps? Or do you think it would be better just to move the new fields to the end?